code/part-one/README.md

```diff
@@ -172,12 +174,13 @@ hash from your first block, and generate a new hash. Rinse and repeat.
 ]
 ```
 
-Now all of our data is linked to our previous data, all the way back to our
-original "genesis block", which is the only block without a previous hash.
-Anyone attempting to modify one block, would also have to modify _every_ block
-that came afterward it in the chain.
+Now all of our data is linked all the way back to our original genesis block.
+If anyone attempts to tamper with the data in a block, it will change the hash
+that gets generated. Anyone checking the chain of hashes would immediately see
```

Comment:
**Avoid "it will":** If anyone attempts to tamper with the data in a block, ~it will change~ the hash that gets generated `also changes`.